test/fixtures/codemod/latest-node/my-app/app/templates/application.hbs

```diff
@@ -1,5 +1,5 @@
-{{!-- The following component displays Ember's default welcome message. --}}
+{{! The following component displays Ember's default welcome message. }}
 <WelcomePage />
-{{!-- Feel free to remove this! --}}
-
-{{outlet}}
\ No newline at end of file
+{{! Feel free to remove this! }}
+{{outlet}}
```

Comment:
There should be a newline both just before and just after `{{outlet}}` (based on the input template in `test/fixtures/codemod/local/my-app/app/templates/application.hbs`)

Reply:
These were remnants from when the codemod did a lot more formatting, and the expected was altered to match. This has been reverted now.